Envfile

```diff
@@ -75,9 +75,9 @@ variable :HONEYBADGER_JS_API_KEY, :String, default: "Optional"
 # Algolia for search (not optional)
 # (https://www.algolia.com/doc/)
 only_in_test = proc { ENV["RACK_ENV"] == "test" ? "test-test" : nil }
-variable :ALGOLIASEARCH_API_KEY, :String, default: only_in_test
-variable :ALGOLIASEARCH_APPLICATION_ID, :String, default: only_in_test
-variable :ALGOLIASEARCH_SEARCH_ONLY_KEY, :String, default: only_in_test
+variable :ALGOLIASEARCH_API_KEY, :String, default: "eef2dc5d0df9438d38080e613623dd1c"
```

Comment:
I don't think you meant to commit this 🙃

Reply:
sorry haha